tgui/packages/tgui/interfaces/SparringContract.tsx

```diff
@@ -70,8 +70,27 @@ export const SparringContract = (props, context) => {
           <Stack vertical fill>
             <Stack.Item>
               <Stack vertical>
-                <Stack.Item fontSize="16px">
-                  Weapons:
+                <Stack.Item>
+                  <Stack fill>
+                    <Stack.Item grow fontSize="16px">
+                      Weapons:
+                    </Stack.Item>
+                    <Stack.Item>
+                      <Button
+                        tooltip={multiline`
+                        The Chaplain's Deity wishes for honorable fighting.
+                        As such, it uses contracts. Signing your name will
+                        set the terms for the battle. Then, the person you
+                        intend to spar with must sign the other side. If terms
+                        are changed on an already signed contract, the
+                        signatures will erase and the new terms must be
+                        re-agreed upon.
+                        `}
+                        icon="info">
+                        Contract?
```

Comment:
might just want to have it as an info icon

Reply:
@tralezab ?

Reply:
i think it's a bit better with contract, just a style choice here

Reply:
could you post pictures of what it looks like? I'm thinking it'll just look weird with both the icon and the text.